src/service/world-api-adapter.js

```diff
@@ -1,11 +1,22 @@
 /**
  * ##World API Adapter
  *
- * The World API Adapter allows you to create, access, and manipulate multiplayer worlds its users and runs
+ * Epicenter includes support for multiplayer games. These are projects where multiple end users share a run and work together in the same "world." Only [team projects](../../../glossary/#team) can be multiplayer.
```

Comment:
Phrasing is slightly confusing here. I think it's do to with too many terms at ones - . i.e., in the introduction for the _world_ adapter the first line references _games_ while the second says these are *projects*)

Maybe define a world first, then explain that worlds are very useful for creating multiplayer games and then talk about our support?